plenary/plenary-27.yaml

```diff
@@ -0,0 +1,39 @@
+metadata:
+  title: Resolutions from the 27th plenary meeting of ISO/TC 154
+  date: 2019-10-17
+  source: Secretariat
+resolutions:
+  - dates:
+      - 2010-05-04
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "ISO 15000-5"
+    identifier: 302
+    
+    actions:
+      - type: resolves
+        date_effective: 2009-09-25
+        message: ISO/TC 154 resolves to ask ISO to publish ISO 15000-5 including corrigenda as approved at the TC154 meeting in 2008-11 (Brussels).
+        
+        
+  - dates:
+      - 2010-05-04
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Appreciation"
+    identifier: 303
+
+    actions:
+      - type: thanks
+        degree: unanimous
+        message: ISO/TC 154 expresses its sincere appreciation to Mr. Wakaizumi for his contribution to the TDED/EDED Alignment project.
+
+
+  - dates:
+      - 2010-05-04
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Appreciation"
+    identifier: 304
+
+    actions:
+      - type: thanks
+        degree: unanimous
+        message: ISO/TC 154 expresses its sincere appreciation to Mr. Van den Akker for his excellent support and dedication.
```

Comment:
In this plenary, I didn't put an `approval` key value in any resolution. Is it right?

Original source capture:
![capture5](https://user-images.githubusercontent.com/33627611/71758993-0cdfba80-2e7d-11ea-9af5-6ae9d6ef0c0f.JPG)

Reply:
I think it should be fine.